EventsExpress/Controllers/AuthenticationController.cs

```diff
@@ -41,27 +54,23 @@ IAuthService authSrv
         [AllowAnonymous]
         [HttpPost("[action]")]
         [Produces("application/json")]
-        public IActionResult Login(LoginDto authRequest)
+        public async Task<IActionResult> Login(LoginDto authRequest)
         {
             if (!ModelState.IsValid)
             {
                 return BadRequest(ModelState);
             }
-
-            var result = _authService.Authenticate(authRequest.Email, authRequest.Password);
-            if (!result.Successed)
+            var (opResult, authResponseModel) = await _authService.Authenticate(authRequest.Email, authRequest.Password);
```

Comment:
Just FYI - we can avoid tuples if AuthResponce will inherit OperationDetails

Reply:
ok, this time I leave here tuple.